fiat-core/src/main/java/com/netflix/spinnaker/fiat/model/resources/Account.java

```diff
@@ -46,9 +48,13 @@ public View getView(Set<Role> userRoles) {
     String name;
     Set<Authorization> authorizations;
 
-    public View(Account account, Set<Role> userRoles) {
+    public View(Account account, Set<Role> userRoles, boolean isAdmin) {
       this.name = account.name;
-      this.authorizations = account.permissions.getAuthorizations(userRoles);
+      if (isAdmin) {
```

Comment:
This if/else in here feels somewhat wrong 🤔 any idea of a better place?

Reply:
Not off the top of my head. I agree that passing a flag around isn't great, but I don't have a better idea.